src/Helpers.js

```diff
@@ -2701,3 +2702,7 @@ export function useDebounce(value, delay) {
   );
   return debouncedValue;
 }
+
+export function isDevelopment() {
+  return !window.location.origin?.includes("gmx.io");
```

Comment:
just curious why using `origin` here instead of `hostname`? both work, origin contains protocol as well